BZ 2110308: Add validation of nodepool name uniqness

rawagner opened the pull request:
Inputs ok
![Screenshot from 2022-08-05 10-21-16](https://user-images.githubusercontent.com/2078045/183034837-bdda0f2f-966e-4996-bc68-13b77825e347.png)

Inputs error
![Screenshot from 2022-08-05 10-14-27](https://user-images.githubusercontent.com/2078045/183034527-4886bfe8-c0ba-4aac-b25a-0a4c55d2212c.png)
![Screenshot from 2022-08-05 10-17-36](https://user-images.githubusercontent.com/2078045/183034550-656ebda7-df42-402f-a16b-0a7ff6f5666d.png)
![Screenshot from 2022-08-05 10-14-27](https://user-images.githubusercontent.com/2078045/183034658-e9de6345-2a25-4841-a0b7-ab07f70b0349.png)

@nirfarkas can you please help with the positioning of the expand/error mgs/labels/edit btn ? In case of error status it seems  kind of messy

jtomasek reviewed (dismissed):
Do we need to also validate the nodepool names are unique across the hub cluster?

rawagner commented:
> Do we need to also validate the nodepool names are unique across the hub cluster?

We should, I've updated the PR and added one for the ACM too https://github.com/stolostron/console/pull/1982

jtomasek reviewed (approved):
Nice!
